.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: erlang
+addons:
+  apt:
+    packages:
+      - libprotobuf-dev
+      - protobuf-compiler
+otp_release:
+  - 19.1
+  - 18.3
+  - 17.5
+  - R16B03
+  - R15B03
+env:
+  - VERBOSE=1
```

Comment:
This isn't strictly necessary, but Travis will time out after 10 minutes if no output is echoed and `VERBOSE` output will prevent that.